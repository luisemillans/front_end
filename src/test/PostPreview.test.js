import React from 'react';
import {shallow} from 'enzyme';
import PostPreview from '../components/PostPreview';
import { italic } from 'ansi-colors';

describe ("<PsotPreview/>",()=>{
    it("Simple Render Works", () =>{
        const component =shallow(<PostPreview/>);
        expect (component).toMatchSnapshot();
    })

    it("Render With props Works", () =>{
        const component = shallow(
            <PostPreview _id={"64132131"} title={"post prueba"}/>
        )

        expect (component.find('.post-title').text()).toBe("post prueba")
    });
})
