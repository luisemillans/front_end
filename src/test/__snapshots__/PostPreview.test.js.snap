// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PsotPreview/> Simple Render Works 1`] = `ShallowWrapper {}`;
